shared_model/validators/query_validator.hpp

```diff
@@ -169,13 +172,16 @@ namespace shared_model {
      */
     template <typename FieldValidator, typename QueryFieldValidator>
     class QueryValidator : public AbstractValidator<interface::Query> {
-     public:
-      QueryValidator(const FieldValidator &field_validator = FieldValidator(),
-                     const QueryFieldValidator &query_field_validator =
-                         QueryFieldValidator())
+      QueryValidator(const FieldValidator &field_validator,
+                     const QueryFieldValidator &query_field_validator)
           : field_validator_(field_validator),
             query_field_validator_(query_field_validator) {}
 
+     public:
+      QueryValidator(std::shared_ptr<ValidatorsConfig> config)
```

Comment:
please check if the config is indeed needed here, as `FieldValidator` seems not needing it

Reply:
IR-439